internal/keepers/ocr.go

```diff
@@ -227,7 +227,7 @@ func (k *keepers) ShouldTransmitAcceptedReport(_ context.Context, rt types.Repor
 		// (while others may not have), don't try to transmit again
 		// TODO: reevaluate this assumption
 		if transmitConfirmed {
-			k.logger.Printf("not transmitting report because upkeep '%s' was already transmitted: %s", string(id.Key), lCtx)
+			k.logger.Printf("not transmitting report because upkeep '%s' was already transmitted: %s", id.Key.String(), lCtx)
```

Comment:
You don't need to do .String(), it would be done automatically if the type implements stringer interface